docs/recipes/README.md

```diff
@@ -1,9 +1,14 @@
 # Recipes
 
 ## CI configurations
-- [CircleCI 2.0 workflows](circleci-workflows.md)
-- [Travis CI](travis.md)
-- [Travis CI with build stages](travis-build-stages.md)
-- [GitLab CI](gitlab-ci.md)
+
+* [CircleCI 2.0 workflows](circleci-workflows.md)
+* [Travis CI](travis.md)
+* [Travis CI with build stages](travis-build-stages.md)
+* [GitLab CI](gitlab-ci.md)
 
 ## Package managers and languages
+
```

Comment:
Can you add these plugins in [plugins-list.md#community-plugins](https://github.com/semantic-release/semantic-release/blob/caribou/docs/extending/plugins-list.md#community-plugins) instead?

Reply:
So this section is reserved for "blessed" plugins? Why hide community plugins away on a different page?

Reply:
This is a section to list all plugins available, from `@semantic-release` org and community.

There no such thing as "blessed" plugins. There is the plugins maintained by the semantic-release team (for which bug report have to be opened in the semantic-release orga for example) and the community plugins that are not maintained by the semantic-release team.
Anyone can make a PR to add his plugin to the list.

I don't know what you mean by "hide". This page is clearly visible, linked in the README and in the sidemenu of the doc website. This is meant to be a registry of plugins.

Reply:
So to be clear, you would want the PHP/Composer link to stay here but the VSCE and Docker links to be moved? Imo that is not very user-oriented, because as a user I just want a single place where I can find out how to get semantic-release working with my package manager

Reply:
we need a clear distinction between plugins that are supported by the semantic-release team and plugins that are supported by the community, to set the right expectations in terms of compatibility and support. I agree that as a user I’d prefer all plugins to be working with the latest semantic-release and be well maintained and supported, but that will not happen and we have to make which plugins are core and which are not.

Once there are more plugins, I’m sure we can build better ways for discoverability, but for starting out that is the best we can do :)

Reply:
How about adding "(community plugin)" after the community plugins in this list?
FWIW gulp highlights almost exclusively community plugins in their recipes.